src/feature/mod.rs

```diff
@@ -283,16 +283,44 @@ pub fn create(trans: &postgres::transaction::Transaction, schema: &Option<valico
                         deltas = array_append(geo.deltas, COALESCE($3, currval('deltas_id_seq')::BIGINT))
                 RETURNING id;
         ", &[&geom_str, &props_str, &delta, &key]) {
-            Ok(res) => Ok(Response {
-                old: id,
-                new: Some(res.get(0).get(0)),
-                version: Some(1)
-            }),
+            Ok(res) => {
+                let new: i64 = res.get(0).get(0);
+                match trans.query("
+                    INSERT INTO geo_history (version, action, geom, props, delta, key, id)
+                        VALUES (
+                            1,
```

Comment:
can we just remove the `key` unique value from the geo_history column? As this doesn't/shouldn't be unique on the geo_history column (I realize I probably had it set when I created it based on the geo table)

Reply:
Addressed in https://github.com/mapbox/Hecate/pull/206/commits/e1baff273e876d047cdc788d7aa4dcf4c950463f